currencies.php

```diff
@@ -0,0 +1,574 @@
+<?php
```

Comment:
Can we move this file to the **inc** directory and rename it **class-camptix-currency.php**? That would be more consistent with the naming convention this plugin already has, and keep the root plugin directory less cluttered.